lib/plugins/aws/package/compile/events/alb/lib/validate.test.js

```diff
@@ -426,4 +426,53 @@ describe('#validate()', () => {
       );
     });
   });
+
+  describe('#validateEventHealthCheck', () => {
```

Comment:
Let's configure tests using `runServerless` util. See documentation on that: https://github.com/serverless/serverless/blob/master/tests/README.md#unit-tests

Reply:
At the moment the only alb test file using `runServerless` is its `index.test.js` file. Given the `runServerless` requirement, perhaps I should be writing tests to `index.test.js` rather than these apparently legacy, lower level tests files?

Reply:
Yes, it's totally fine if you add them to `index.test.js`or you may also create another one `healthCheck.test.js`